bin/update-patterns

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+# Updates local copy of the Libero patterns (in vendor-extra/LiberoPatternsBundle/src/Resources)
+#
+# Clones and builds the Pattern Library (https://github.com/libero/pattern-library), then copies the CSS and fonts.
+
+function finish {
+    docker rm browser-pattern-library &> /dev/null || true
+}
+
+trap finish EXIT
+
+cd "$(dirname "$0")/.."
+
+mkdir -p var/pattern-library && cd "$_"
+
+if ! [[ -d .git ]]; then
+    git clone git@github.com:libero/pattern-library.git .
+else
+    git reset --hard origin/master
```

Comment:
is there a potential for this script to work on a branch of `pattern-library`?

Reply:
Not particularly, only running the build and export commands which isn’t much duplication. Plus this should be replaced by using ready-built artifacts anyway.